src/fairness/weighted_tree/Makefile.am

```diff
@@ -13,21 +13,27 @@ noinst_HEADERS = \
     ../account/account.hpp \
     weighted_tree.hpp \
     weighted_walk.hpp \
+	../reader/data_reader_base.hpp \
```

Comment:
Looks like a mixing of tabs and spaces

Reply:
Oops. Thanks for pointing that out. I just force-pushed a change to fix that.